yarn-project/p2p/src/service/discv5_service.test.ts

```diff
@@ -48,7 +61,8 @@ describe('Discv5Service', () => {
     const node2 = await createNode(port);
     await node1.start();
     await node2.start();
-    await sleep(200);
+    await waitForPeers(node2, 2, 4000);
```

Comment:
I feel like it's probably simpler to just removed the 3rd argument here and use the default. It means fewer hardcoded number littered around